src/actions/emojis.js

```diff
@@ -79,13 +83,13 @@ export function getCustomEmojiByName(name) {
             {
                 type: EmojiTypes.GET_CUSTOM_EMOJI_SUCCESS
             }
-        ]));
+        ]), getState);
```

Comment:
Flow again, I suppose? Does this basically mean optional parameters are always required?

Reply:
No, we can make parameters optional, I didn't done it, but this is a good catch. :)

Reply:
Changed! Thanks! :)